recipes/towhee/meta.yaml

```diff
@@ -0,0 +1,115 @@
+{% set name = "towhee" %}
+{% set version = "0.4.0rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aa6d6996c9a48b47be31b6aa1279d3e7f738e4aa12ebfecd4d5cf8c5d40ee1e6
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - pyyaml >=5.3.0
+    - requests >=2.12.5
+    - tqdm >=4.59.0
+
+test:
+  imports:
+    - towhee
+    - towhee.array
+    - towhee.cnn_trainer
+    - towhee.cnn_trainer.utils
+    - towhee.dag
+    - towhee.dag.utils
+    - towhee.data_source
+    - towhee.dataframe
+    - towhee.engine
+    - towhee.engine.operator_io
+    - towhee.engine.operator_runner
+    - towhee.hub
+    - towhee.models
+    - towhee.models.layers
+    - towhee.models.layers.activations
+    - towhee.models.loss
+    - towhee.models.retina_face
+    - towhee.models.swin_transformer
+    - towhee.models.utils
+    - towhee.operator
+    - towhee.tests
+    - towhee.tests.array
+    - towhee.tests.cnn_trainer
+    - towhee.tests.dag
+    - towhee.tests.dag.utils
+    - towhee.tests.dataframe
+    - towhee.tests.engine
+    - towhee.tests.engine.operator_io
+    - towhee.tests.engine.operator_runner
+    - towhee.tests.hub
+    - towhee.tests.mock_operators
+    - towhee.tests.mock_operators.add_operator
+    - towhee.tests.mock_operators.cv2_decoder
+    - towhee.tests.mock_operators.flat_operator
+    - towhee.tests.mock_operators.generator_operator
+    - towhee.tests.mock_operators.norm_data_operator
+    - towhee.tests.mock_operators.pytorch_cnn_operator
+    - towhee.tests.mock_operators.pytorch_image_classification_operator
+    - towhee.tests.mock_operators.pytorch_object_detection_operator
+    - towhee.tests.mock_operators.pytorch_transform_operator
+    - towhee.tests.mock_operators.pytorch_transformer_operator
+    - towhee.tests.mock_operators.repeat_operator
+    - towhee.tests.mock_operators.sub_operator
+    - towhee.tests.mock_operators.sum_operator
+    - towhee.tests.mock_operators.zero_drop
+    - towhee.tests.models
+    - towhee.tests.models.layers
+    - towhee.tests.models.loss
+    - towhee.tests.models.swin_transformer
+    - towhee.tests.models.utils
+    - towhee.tests.operators
+    - towhee.tests.test_util
+    - towhee.tests.trainer
+    - towhee.tests.utils
+    - towhee.trainer
+    - towhee.trainer.models
+    - towhee.trainer.models.layers
+    - towhee.trainer.models.swin_transformer
+    - towhee.trainer.models.utils
+    - towhee.trainer.optimization
+    - towhee.utils
+  commands:
+    - pip check
+  requires:
+    - pip
+    - pytorch >=1.2.0
```

Comment:
I deliberately add it here for the package name in conda-forge (which is pytorch) is different with the one provided by pip (which is torch).

Reply:
If this is required, it's better if you add it to `run`. The difference in names is not a problem, the package is the same.

Reply:
This still needs to be adressed.

Reply:
> This still needs to be adressed.

Should I add it only to `run` or both `run` & `test.requires`?

Reply:
run is sufficient